src/utils/coordinator.js

```diff
@@ -0,0 +1,64 @@
+import { DELAY_TO_NEXT_FORGER } from '../constants'
+
+/**
+ * Extracts the next forgers without duplicates from the coordinator state returned by
+ * the Hermez API
+ * @param {Object} coordinatorState - Coordinator state returned by the Hermez API
+ * @returns Next forgers
+ */
+function getNextForgers (coordinatorState) {
```

Comment:
Another way to do this is with a Set, it removes duplicates

Reply:
Yep, but it doesn't work with objects, it only takes into account their reference instead of their values to create the set